### PR TITLE
Adds AmCharts

### PR DESCRIPTION
**Pull request by mlzummo**
@GordonSmith @jbrundage @dtsnell4 Please Review
Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -38,8 +98,67 @@ if (window.location.hostname === "rawgit.com") {
...
+            'amcharts.gauge': "http://rawgit.com/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/gauge",
+            'amcharts.pie': "http://rawgit.com/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/pie",
+            'amcharts.radar': "http://rawgit.com/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/radar",
+            'amcharts.serial': "http://rawgit.com/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/serial",
+            'amcharts.xy': "http://rawgit.com/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/xy",
+            'amcharts.plugins.responsive': "http://rawgit.com/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/plugins/responsive/responsive",
+            'amcharts.plugins.dataloader': "http://rawgit.com/arturgspb/bower-amcharts/v3.13.3/dist/amcharts/plugins/dataloader",
             "src": "../src"
+        },
+    shim: {
```
Comment (GordonSmith) [inline src/config.js]: I am 90% sure the shim is not needed here.

Comment (mlzummo) [inline src/config.js]: without the shims there were errors

File: src/config.js [hunk 2]
```diff
@@ -16,8 +16,68 @@ require.config({
...
+            }
+        },
+        'amcharts.xy': {
+            deps: ['amcharts','amcharts.plugins.responsive'],
+            exports: 'AmCharts',
+            init: function() {
+                AmCharts.isReady = true;
+            }
+        },
+        'amcharts.plugins.responsive': {
```
Comment (GordonSmith) [inline src/config.js]: Is this shim needed?

File: src/config.js [hunk 3]
```diff
@@ -16,8 +16,68 @@ require.config({
...
+            deps: [ 'amcharts' ]
+        },
+        'amcharts.plugins.dataloader': {
```
Comment (GordonSmith) [inline src/config.js]: Is this shim needed?

File: src/config.js [hunk 4]
```diff
@@ -16,8 +16,68 @@ require.config({
...
+        'amcharts.serial': "../bower_components/amcharts/dist/amcharts/serial",
+        'amcharts.xy': "../bower_components/amcharts/dist/amcharts/xy",
+        'amcharts.plugins.responsive': "../bower_components/amcharts/dist/amcharts/plugins/responsive/responsive",
+        'amcharts.plugins.dataloader': "../bower_components/amcharts/dist/amcharts/plugins/dataloader",
 
         "src": "../src"
+    },
+    shim: {
+        'amcharts.funnel': {
+            deps: ['amcharts','amcharts.plugins.responsive'],
```
Comment (GordonSmith) [inline src/config.js]: Whats the responsive bit?  (and is it needed?)

File: src/config.js (hunk 3 above)
Comment (mlzummo) [inline src/config.js]: responsive perhaps, dataloader i dont think so... responsive is suppose to help charts work better in responsive env,

**Comment by GordonSmith** [comment]
I have made some changes, primarily to make it build:  https://github.com/mlzummo/Visualization/pull/2

**Comment by GordonSmith** [comment]
Fixed #318

**Pull request merged**